dynamics-nav/Preserving-Data-When-Working-with-a-Statically-Generated-Proxy.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: "Preserving Data When Working with a Statically Generated Proxy"
+title: "Preserving Data and Avoiding Data Loss"
```

Comment:
'Preserving Data' and 'Avoiding Data Loss' sound like the same thing to me. I can't think of a better title, though.